ERROR: Io(Custom { kind: InvalidData, error: AlertReceived(CertificateUnknown) })
Error: failed to connect Cumulocity cloud.

Caused by:
    Connection check failed
